src/main/java/frc/robot/Robot.java

```diff
@@ -139,23 +144,26 @@ public void disabledInit() {
     drivetrain.brakeMode();
   }
 
-  // @Override
-  // public void disabledPeriodic() {
-  //   String name = m_robotContainer.getAutoName(m_robotContainer.getAuto());
-
-  //   if (name != "Nothing?????/?///?") {
-  //     Trajectory accumulator = new Trajectory();
-  //     List<PathPlannerPath> paths = PathPlannerAuto.getPathGroupFromAutoFile(name);
-  //     for (int i = 0; i < paths.size(); i++) {
-  //       accumulator =
-  //           accumulator.concatenate(
-  //               pathTrajToTragTraj(
-  //                   paths.get(i).getTrajectory(new ChassisSpeeds(), new Rotation2d())));
-  //     }
-  //     field.getObject("traj").setTrajectory(accumulator);
-  //     SmartDashboard.putData(field);
-  //   }
-  // }
+  String lastName = "";
+
+  @Override
+  public void disabledPeriodic() {
+    String name = m_robotContainer.getAutoName(m_robotContainer.getAuto());
+
+    if (name != lastName && name != "Nothing?????/?///?") {
```

Comment:
Be careful here!   You're not comparing primitives, so how should you do it instead?

Reply:
equals now yay